detection_rules/etc/version.lock.json

```
@@ -6590,7 +6688,7 @@
     "rule_name": "Suspicious WMI Event Subscription Created",
     "sha256": "bee333bfc8d77b96f009283d0b8dc93b5e2e38ef6b27b38b21daccf6fe50833a",
     "type": "eql",
-    "version": 4
+    "version": 6
```

```suggestion
    "version": 6
```

unexpected double bump, taking a look